### PR TITLE
style(app): hs calibration module cal fixes

### PR DESCRIPTION
**Pull request by mjhuff**
Closes RQA-1932, RQA-1934

# Overview

This PR fixes a few style issues and lightly refactors BeforeBeginning for module cal.

### New Screwdriver for HS

Screenshot 2023-12-12 at 12 55 41 PM

### Flip the Body Text and Warning (with 16px separating the two!)

Screenshot 2023-12-12 at 1 12 03 PM

# Test Plan
- Verify the image exists, the flow looks good, and the equipmentList isn't broken.

# Changelog
- Added a new screwdriver to the required equipment list for the Heater-Shaker.

# Risk assessment
low

### CODE REVIEW

File: app/src/assets/localization/en/heater_shaker.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
   "show_attachment_instructions": "Show attachment instructions",
   "stop_shaking_start_run": "Stop shaking and start run",
   "stop_shaking": "Stop Shaking",
-  "t10_torx_screwdriver": "{{name}} Screwdriver",
+  "t10_torx_screwdriver": "T10 Torx Screwdriver",
```
Comment (jerader) [inline app/src/assets/localization/en/heater_shaker.json]: T10 Torx is a proper noun and shouldn't be in the i18n string right? I always forget what our best practice is for proper nouns.

File: app/src/organisms/ModuleWizardFlows/BeforeBeginning.tsx [hunk 2]
```diff
@@ -47,8 +57,9 @@ export const BeforeBeginning = (
   }, [])
   const moduleDisplayName = getModuleDisplayName(attachedModule.moduleModel)
 
-  let adapterLoadname
-  let adapterDisplaynameKey
+  let adapterLoadname: string
+  let adapterDisplaynameKey: string
```
Comment (jerader) [inline app/src/organisms/ModuleWizardFlows/BeforeBeginning.tsx:61]: 🖌️

File: app/src/organisms/ModuleWizardFlows/BeforeBeginning.tsx [hunk 3]
```diff
@@ -102,3 +115,22 @@ export const BeforeBeginning = (
...
+  equipmentList: EqipmentItem[],
+  attachedModule: AttachedModule
+): EqipmentItem[] => {
+  const { t } = useTranslation('heater_shaker')
+  if (
+    HEATERSHAKER_MODULE_MODELS.some(
+      model => model === attachedModule.moduleModel
+    )
+  ) {
+    equipmentList.unshift({
```
Comment (jerader) [inline app/src/organisms/ModuleWizardFlows/BeforeBeginning.tsx:129]: wow i have never used `unshift` before!

**Review by jerader — APPROVED** [review]
nice! makes sense to me. Don't forget to change the PR title to be pydantic

File: app/src/assets/localization/en/heater_shaker.json (hunk 1 above)
Comment (mjhuff) [reply app/src/assets/localization/en/heater_shaker.json]: Ahhhh, that totally makes sense. Good to know for the future, and I'll update it now!

**Pull request merged**